Makefile

```diff
@@ -1,6 +1,39 @@
 FILENAME=template
+EMACS=emacs
 
-all:
+all: README.org $(FILENAME).html
+
+# Original markdown targets are here.
+
+$(FILENAME).html: README.md
 	ln -sf README.md $(FILENAME).md
 	pandoc --from=gfm README.md -o $(FILENAME).html
+
+$(FILENAME).pdf:
 	pandoc --standalone $(FILENAME).html -o $(FILENAME).pdf
+
+# New org-mode targets are here.
+
+README.org: template.scm
```

Comment:
Please drop the code for generating `README.org` and `index.html`.  The `srfi generate-srfi` command already takes care of that.  Also, that's a job for the editor.  Authors don't need to do that.